data-tool/flows/migrate_corps_flow.py

```diff
@@ -324,40 +391,89 @@ def load_event_filing_data(config, app: any, colin_db_engine: engine, db_lear, e
 
 
 
-# @flow(name="Corps-Migrate-ETL", task_runner=ConcurrentTaskRunner())
-# @flow(name="Corps-Migrate-ETL", task_runner=DaskTaskRunner())
-@flow(name="Corps-Migrate-ETL", task_runner=SequentialTaskRunner())
+@flow(
+    name="Corps-Migrate-ETL",
+    description="Migrate corporation data through ETL process",
+    version="1.0",
+    log_prints=True
+)
 def migrate_flow():
     # setup
     config = get_config()
     db_colin_engine = colin_init(config)
     FLASK_APP, db_lear = lear_init(config)
 
     unprocessed_corps = get_unprocessed_corps(config, db_colin_engine)
+    print(f"Found {len(unprocessed_corps)} corps to process")
+
+    # Submit all event filing tasks in parallel
+    event_filing_futures = []
+    for corp in unprocessed_corps:
+        future = get_event_filing_data.submit(
+            config=config,
+            colin_db_engine=db_colin_engine,
+            unprocessed_corp_dict=corp
+        )
+        event_filing_futures.append(future)
+
+    # Process results and submit cleaning tasks
+    clean_futures = []
+    for future in event_filing_futures:
+        data = future.result()  # Wait for result
+        if data:
+            clean_future = clean_event_filing_data.submit(
+                config=config,
+                colin_db_engine=db_colin_engine,
+                event_filing_data_dict=data
+            )
+            clean_futures.append(clean_future)
+
+    # Process cleaning results and submit transform tasks
+    transform_futures = []
+    for future in clean_futures:
+        data = future.result()  # Wait for result
+        if data:
+            transform_future = transform_event_filing_data.submit(
+                config=config,
+                colin_db_engine=db_colin_engine,
+                event_filing_data_dict=data
+            )
+            transform_futures.append(transform_future)
+
+    # Process transform results and submit load tasks
+    load_futures = []
+    for future in transform_futures:
+        data = future.result()  # Wait for result
+        if data:
+            load_future = load_event_filing_data.submit(
+                config=config,
+                app=FLASK_APP,
+                colin_db_engine=db_colin_engine,
+                db_lear=db_lear,
+                event_filing_data_dict=data
+            )
+            load_futures.append(load_future)
+
+    # Wait for all loads to complete
+    results = [future.result() for future in load_futures]
+
+    # Summarize results
+    successful = sum(1 for r in results if r is not None)
+    failed = len(results) - successful
+    print(f"Processing complete. Successful: {successful}, Failed: {failed}")
```

Comment:
wondering...is the results for successfully completed tasks always not `None` when you tested it as the `load_event_filing_data()` doesn't return value explicitly?

I use these in my pr
```python
wait(delete_futures)
succeeded = sum(1 for f in delete_futures if f.state.is_completed())
failed = len(delete_futures) - succeeded
```

Reply:
yeah that didn't work properly.  it only printed out when all corps migrated successfully.

For now I've just removed the code altogether.  I think prefect UI and the tracking table can give us enough info.